gradle.properties

```diff
@@ -5,4 +5,5 @@ junit_jupiter_api_version=5.5.2
 junit_jupiter_engine_version=5.5.2
 logback_version=1.2.3
 selenide_version=5.15.0
-gson_version=2.8.6
\ No newline at end of file
+gson_version=2.8.6
+assertj_version3.6.1
```

Comment:
incorrect syntax, the build would fail
To avoid this, always do a build before you push the code.